src/components/ActionsMenuLayout/ActionsMenuLayout.driver.ts

```diff
@@ -30,6 +31,9 @@ export const actionsMenuLayoutDriverFactory = (
       (await base.attr(ACTIONS_MENU_DATA_KEYS.mobile)) === 'true',
     item: (content) => actionsMenuLayoutItem(base, content),
     clickItem: (content) => actionsMenuLayoutItem(base, content).click(),
+    isItemFocused: async (content) =>
+      (await actionsMenuLayoutItem(base, content).getNative()) ===
+      document.activeElement,
```

Comment:
Note that this won't work in e2e tests, since the `document` doesn't exist.
Unfortunately this type of logic needs to be moved to the test itself.

Reply:
ah, i guess thats why it was like this in the first place. will update